test/cljs/rems/administration/test_create_form.cljs

```diff
@@ -353,7 +367,9 @@
                                                                  :label {:en "Bacon"
                                                                          :fi "Pekonia"}}]))]
         (is (= {:organization {:organization/id "abc"}
-                :form/title "the title"
+                :form/internal-name "the name"
+                :form/external-title {:en "en external title"
+                                      :fi "fi external title"}
```

Comment:
should there be a test case here for a missing external-title?

Reply:
Not sure if my test makes any sense :shrug:

Reply:
well at least the behaviour is now explicit